pkg/operator/operator.go

```diff
@@ -89,6 +90,13 @@ func New(config operatorconfig.Config, kubeConfig *rest.Config) (*Operator, erro
 		return nil, fmt.Errorf("failed to create ingress controller: %v", err)
 	}
 
+	// Create and register the ingress routes controller with the operator manager.
+	if _, err := ingressroutescontroller.New(mgr, ingressroutescontroller.Config{
+		SecretNamespace: "openshift-config",
```

Comment:
the `openshift-config` namespace is already defined in `pkg/operator/controller/names.go` as `GlobalMachineSpecifiedConfigNamespace`.

Reply:
[It looks like](https://github.com/awgreene/cluster-ingress-operator/blob/master/pkg/operator/controller/names.go#L16) the `GlobalMachineSpecifiedConfigNamespace` is actually `openshift-config-managed`.

Reply:
I can add a new constant to the `names.go` file though.

Reply:
Ah, my mistake (so close)! If you could add a new constant that would be great! thanks.